seeds/04_applications.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable camelcase, max-len */
+'use strict';
+
+exports.seed = function(knex) {
+  return knex('application').del()
+    .then(() => {
+      return knex('application').insert([{
+        id: 1,
+        user_id: 1,
+        job_id: 1,
+        status: 'applied',
+        role: 'Software Engineer',
+        rating: 5,
+        note: 'This job pays extremely well',
+        created_at: new Date('2017-02-02 14:26:16 UTC'),
+        updated_at: new Date('2017-02-02 14:26:16 UTC')
+      },
+      {
+        id: 2,
+        user_id: 1,
+        job_id: 2,
+        status: 'have not applied',
+        role: 'Front-end Engineer',
+        rating: 4,
+        note: 'The company culture seems like a great fit',
+        created_at: new Date('2017-02-02 14:26:16 UTC'),
+        updated_at: new Date('2017-02-02 14:26:16 UTC')
+      },
+      {
+        id: 3,
```

Comment:
Applications 3 & 4 are almost identical to 1 & 2. Maybe just change the role and the note on them so that when we are building things out we can differentiate the output and ensure we are getting the records we expect.